.circleci/config.yml

```diff
@@ -581,7 +581,40 @@ jobs:
       - *setup_env
       - run:
           name: "Build and test"
-          command: force_deploy_build aztec-sandbox
+          command: force_deploy_build aztec-sandbox-base false
+
+  aztec-sandbox-x86_64:
+    machine:
+      image: ubuntu-2004:202010-01
+    resource_class: large
+    steps:
+      - *checkout
+      - *setup_env
+      - run:
+          name: "Build and test"
+          command: force_deploy_build aztec-sandbox true x86_64
```

Comment:
we want to be re-building these on every commit?

Reply:
Good spot. No just the same rules as aztec-sandbox-base.